src/mgr/ActivePyModules.cc

```diff
@@ -165,6 +166,9 @@ PyObject *ActivePyModules::get_daemon_status_python(
 
 PyObject *ActivePyModules::get_python(const std::string &what)
 {
+  /* INJECT */
+  PyObject *injected_map = Injector::get_python(what);
```

Comment:
If we were to add the fake maps what would be best, keep `Injector::get_python` in ActivePyModules as now but store the osdmap in `ClusterState` or, modify the `ClusterState::with_osdmap` so the `ActivePyModules` is decoupled from the Injection logic?

Reply:
the latter would be best since it would cover all the uses in ActivePyModules as well - ```ClusterState::with_osdmap``` and ```ClusterState::with_osdmap_and_pgmap```should cover all users in the mgr (I may have missed some, but that's the vast majority if not all of them)

Reply:
done in 6de8074